lib/rototiller/task/params/command.rb

```diff
@@ -165,19 +130,34 @@ def add_argument(*args, &block)
         end
       end
 
+      # convert a Command object to a string safely (redact sensitive env vars)
+      # @return [String] the current value of the command string as built from its params
+      #   if any params values are set from sensitive env vars, redact them
+      # @api public
+      # @example puts command.safe_print
+      # TODO make private method? so that it will throw an error if yielded to?
+      def safe_print
+        if @is_value_sensitive
+          my_value = "[REDACTED]"
+        elsif name
+          my_value = name
+        end
```

Comment:
weird that rubocop didn't catch this.  this should probably be assigned based on the output of the conditional, instead of two possible assignments inside the conditional. 🤷‍♂️